glossary/repair.md

```diff
@@ -1,6 +1,6 @@
 type of [achievable](https://github.com/newatoms/guides/blob/ready/glossary/achievable.md), where you repair something which was broken, i.e. not working the way it should.
 
-*Repairs* are documented by simply stating what is broken and, if applicable, how to replicate the broken thing. 
+*Repairs* are documented by simply stating what is broken and, if applicable, instructions on how to get the broken result. 
```

Comment:
"how to get the broken result" is still unclear. Do you mean "how to resolve the break"?